src/application.ts

```diff
@@ -294,6 +452,97 @@ export class MyriadApiApplication extends BootMixin(
     };
   }
 
+  async upload(data: AnyObject): Promise<string> {
+    const assetPath = path.join(__dirname, `../seed-data/assets`);
+
+    if (!existsSync(assetPath)) return '';
+
+    const files = fs.readdirSync(assetPath);
+    const file = files.find(e => {
+      const [imageFile] = e.split(/\.(jpeg|jpg|gif|png)$/);
+      if (imageFile === data.imageFile) return true;
+      return false;
+    });
+
+    if (!file) return '';
+
+    const tmpDir = os.tmpdir();
+    const filePath = `${assetPath}/${file}`;
+    const formattedFilePath = `${tmpDir}/${file}`;
+    const uploadFilePath = `${data.imagePath}/${file}`;
+
+    await sharp(filePath)
+      .resize({width: 400})
+      .withMetadata()
+      .toFormat('jpg')
+      .toFile(formattedFilePath);
+
+    let result = '';
+    if (config.FIREBASE_STORAGE_BUCKET) {
+      const bucket = firebaseAdmin.storage().bucket();
+      const [imageFile] = await bucket.upload(formattedFilePath, {
+        resumable: false,
+        public: true,
+        destination: uploadFilePath,
+      });
+
+      result = imageFile.publicUrl();
+    } else {
+      if (!config.LOCAL_STORAGE_URL) {
+        throw new Error('Storage Not Found');
+      }
+
+      const folderPath = `../storages/${data.imagePath}`;
+      const storagePath = path.join(__dirname, `${folderPath}`);
+
+      if (!fs.existsSync(storagePath)) {
+        fs.mkdirSync(storagePath, {recursive: true});
+      }
+
+      fs.copyFileSync(formattedFilePath, `${storagePath}/${file}`);
+
+      result = `${config.LOCAL_STORAGE_URL}/storages/${uploadFilePath}`;
+    }
+
+    fs.unlinkSync(formattedFilePath);
+
+    return result;
+  }
+
+  userData(data: AnyObject[]): AnyObject[] {
+    return [
+      ...data,
+      {
+        user: {
+          name: 'Myriad Official',
+          username: 'myriad_official',
+          profilePictureURL:
+            'https://pbs.twimg.com/profile_images/1407599051579617281/-jHXi6y5_400x400.jpg',
```

Comment:
use asset from own storage like network image, ignore `bannerImageUrl` keep it blank